src/utils/components/LazyActionMenu/ActionMenuItem.tsx

```diff
@@ -0,0 +1,120 @@
+/* eslint-disable no-nested-ternary */
```

Comment:
why?

Reply:
The code ported from the Console uses nested ternary. I will rewrite and remove this exception.